config/webpack/plugins/createTreatPlugin.js

```diff
@@ -17,11 +17,9 @@ const braidThemes = [
 module.exports = ({
   target,
   isProductionBuild,
-  include,
```

Comment:
This is still being passed through at the [call site](https://github.com/seek-oss/sku/pull/616/files#diff-4e4806abd6e6dec03e7f50c209f69b659d0ceeca5fae03a88e3c40876b6e1de5L193)